chingu-coders/Voyage2-Turtles-02
# clock persistent settings and sticknote bug fix

## Purpose
* Makes user-selected clock options persist across sessions
* fix stickynote fadein bug

## Approach/Fixes
* Just makes sense

## Testing
* Checkout this PR
* Change the clock settings
* refresh tab and confirm that changes to settings persist
* when app fades in, make sure sticknotes fade in sync with rest of body
